plugins/obs-vst/VSTPlugin.cpp

```diff
@@ -38,6 +38,10 @@ intptr_t VSTPlugin::hostCallback_static(AEffect *effect, int32_t opcode,
 	case audioMasterGetCurrentProcessLevel:
 		return 1;
 
+	case audioMasterGetVendorString:
+		strncpy((char*)ptr, "OBS Studio", 11);
```

Comment:
Code format probably wants a space before the asterisk. Otherwise this looks good to me.

Reply:
thanks @WizardCM !
sorry about that, committed the fix and it looks ok now.